packages/gatsby/src/utils/page-data.ts

```diff
@@ -136,7 +136,19 @@ export async function flush(): Promise<void> {
 
   const { pagePaths } = pendingPageDataWrites
 
-  const pagesToWrite = pagePaths.values()
+  const pagesToWrite = Array.from(pagePaths.values())
```

Comment:
Let's change it to this. It's a Set that already only holds values.

```js
const { pagePaths: pagesToWrite } = pendingPageDataWrites
```

Reply:
ah ok, typescript lied to me and just said it was an "IterableIterator" :-D

Reply:
👍